src/m365/teams/commands/meeting/meeting-add.ts

```diff
@@ -0,0 +1,210 @@
+        recordAutomatically: !!args.options.recordAutomatically
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-s, --startTime [startTime]'
+      },
+      {
+        option: '-e, --endTime [endTime]'
+      },
+      {
+        option: '--subject [subject]'
+      },
+      {
+        option: '-p, --participantUserNames [participantUserNames]'
+      },
+      {
+        option: '--organizerEmail [organizerEmail]'
+      },
+      {
+        option: '-r, --recordAutomatically'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+
+        if (args.options.startTime && !validation.isValidISODateTime(args.options.startTime)) {
+          return `'${args.options.startTime}' is not a valid ISO date string for startTime.`;
+        }
+
+        if (args.options.endTime && !validation.isValidISODateTime(args.options.endTime)) {
+          return `'${args.options.endTime}' is not a valid ISO date string for endTime.`;
+        }
+
+        if (args.options.startTime && args.options.endTime && new Date(args.options.startTime) >= new Date(args.options.endTime)) {
+          return 'The startTime value must be before endTime.';
+        }
+
+        if (args.options.startTime && new Date() >= new Date(args.options.startTime)) {
+          return 'The startTime value must be in the future.';
+        }
+
+        if (args.options.endTime && new Date() >= new Date(args.options.endTime)) {
+          return 'The endTime value must be in the future.';
+        }
+
+        if (args.options.participantUserNames) {
+          const participants = args.options.participantUserNames.trim().toLowerCase().split(',').filter(e => e && e !== '');
+
+          if (!participants || participants.length === 0 || participants.some(e => !validation.isValidUserPrincipalName(e))) {
+            return `${args.options.participantUserNames} contains one or more invalid UPN.`;
+          }
+        }
+
+        if (args.options.organizerEmail && !validation.isValidUserPrincipalName(args.options.organizerEmail)) {
+          return `'${args.options.organizerEmail}' is not a valid email for organizerEmail.`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      const isAppOnlyAccessToken = accessToken.isAppOnlyAccessToken(auth.service.accessTokens[this.resource].accessToken)!;
+
+      if (isAppOnlyAccessToken && !args.options.organizerEmail) {
+        throw `The option 'organizerEmail' is required when creating a meeting using app only permissions`;
+      }
+
+      if (!isAppOnlyAccessToken && args.options.organizerEmail) {
+        throw `The option 'organizerEmail' is not supported when creating a meeting using delegated permissions`;
+      }
+
+      let graphBaseUrl = `${this.resource}/v1.0/`;
```

Comment:
```suggestion
      let requestUrl = `${this.resource}/v1.0/`;
```

Reply:
ok I have changed it, In app-only mode it is reassigned to `requestUrl = `${this.resource}/v1.0/users/${organizerId}`;`